Sources/mpc-core-kit-swift/MpcCoreKitSwift.swift

```diff
@@ -387,6 +387,11 @@ public struct MpcCoreKit  {
         return try await TssModule.get_tss_pub_key(threshold_key: threshold_key, tss_tag: selectedTag)
     }
     
+    public func commitChanges() async throws {
+        // create a copy syncMetadata
+        try await self.tkey?.sync_local_metadata_transistions()
```

Comment:
This should throw if tkey is nil. This behaviour would likely be undefined if tkey is nil.